UGW: During package/install scripts are executed no build host machine.
	 Workaround to untill target scripts under /etc/init.d/* are fixed

diff --git a/package/Makefile b/package/Makefile
--- a/package/Makefile
+++ b/package/Makefile
@@ -100,6 +100,11 @@ PACKAGE_INSTALL_FILES:= \
 		) \
 	)
 
+ifeq ($(CONFIG_INVERSE_PKG_IPKG_INSTALL),)
+PACKAGE_IPKG_NO_SCRIPT:= \
+		IPKG_NO_SCRIPT=1
+endif
+
 $(curdir)/cleanup: $(TMP_DIR)/.build
 	rm -rf $(STAGING_DIR_ROOT)
 
@@ -122,7 +127,7 @@ PACKAGE_INSTALL_FILES:= \
 	@( \
 		cd $(TARGET_DIR); \
 		for script in ./usr/lib/opkg/info/*.postinst; do \
-			IPKG_INSTROOT=$(TARGET_DIR) $$(which bash) $$script; \
+			IPKG_INSTROOT=$(TARGET_DIR) $(PACKAGE_IPKG_NO_SCRIPT) $$(which bash) $$script; \
 		done; \
 		for script in ./etc/init.d/*; do \
 			grep '#!/bin/sh /etc/rc.common' $$script >/dev/null || continue; \
